widget/edit.controller.js

```diff
@@ -203,9 +203,9 @@
               LABEL_FILTER_CRITERIA: widgetUtilityService.translate('socOverviewSankey.LABEL_FILTER_CRITERIA'),
               LABEL: widgetUtilityService.translate('socOverviewSankey.LABEL'),
               LABEL_SOURCE_NODE: widgetUtilityService.translate('socOverviewSankey.LABEL_SOURCE_NODE'),
-              LABEL_SELECT_SOURCE_NODES: widgetUtilityService.translate('socOverviewSankey.LABEL_SELECT_SOURCE_NODES'),
+              LABEL_SELECT_SOURCE_NODE: widgetUtilityService.translate('socOverviewSankey.LABEL_SELECT_SOURCE_NODE'),
```

Comment:
I don't see any HTML file changes where these variables are used. Changes might required in those files too.

Reply:
only changes were not updated in edit controller